recipes/libsndfile/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+import glob
+
+
+class LibnameConan(ConanFile):
+    name = "libsndfile"
+    description = "Libsndfile is a library of C routines for reading and writing files containing sampled audio data."
+    topics = ("conan", "libsndfile", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.mega-nerd.com/libsndfile"
+    license = "LGPL-2.1"
+    generators = 'pkg_config'
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_alsa": [True, False],
+        "with_sqlite": [True, False],
+        "with_external_libs": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_alsa": True,
+        "with_sqlite": True,
+        "with_external_libs": True,
+        "libalsa:shared": True}
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.options.with_alsa:
+            self.requires("libalsa/1.1.9")
+        if self.options.with_sqlite:
+            self.requires("sqlite3/3.29.0")
+        if self.options.with_external_libs:
+            self.requires("flac/1.3.3")
+            self.requires("ogg/1.3.4")
+            self.requires("vorbis/1.3.6")
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("The project libsndfile can not be built by Visual Studio.")
```

Comment:
The release contains a cmake script.
Doesn't that build using MSVC?

Reply:
the current master is probably ok, but at the time of 1.0.28, cmake was very experimental, and build instructions do not cover windows.

Reply:
well, MSVC covers libsndfile, so obviously, it works with MSVC: https://github.com/microsoft/vcpkg/tree/master/ports/libsndfile

Reply:
they are using a specific commit on master, because there were no release since 1.0.28: https://github.com/microsoft/vcpkg/blob/master/ports/libsndfile/portfile.cmake#L3
Do we want to do the same ?

Reply:
I think this pr is fine.
I browsed their issue tracker and it looks like a release is not around the corner.